core/classes/AuthFlags.class.php

```diff
@@ -397,5 +436,31 @@ function getReauthenticationLifetime() {
 
 		return $this->reauthentication_expiry;
 	}
+	
+	/**
+	 * Sets if the auth pugin is capable of provision users
+	 *
+	 * @param bool $p_autoprov True, if the plugin can create users
+	 * @return void
+	 * @see getAutoprovisionCapability()
+	 */
+	function setAutoprovisionCapability( $p_autoprov ) {
```

Comment:
Should we use `setAutoProvisioningCapability` and `getAutoProvisioningCapability`?  Remember to update the php-docs as well.

Reply:
I don't think so, since the config variable in the pluging called 'autoprovision' not 'autoprovisioning'.